ods_ci/tests/Tests/1100__data_science_pipelines/1105__data-science-pipelines-gpu.robot

```diff
@@ -4,46 +4,65 @@ Resource            ../../Resources/RHOSi.resource
+${PROJECT}=    dsp-gpu
+${PIPELINE_NVIDIA_GPU}=    tests/Resources/Files/pipeline-samples/v2/cache-disabled/gpu/pytorch/pytorch_nvidia_gpu_availability_compiled.yaml
+${PIPELINE_AMD_GPU}=    tests/Resources/Files/pipeline-samples/v2/cache-disabled/gpu/pytorch/pytorch_amd_gpu_availability_compiled.yaml
 # robocop: on=line-too-long
 
 
 *** Test Cases ***
 # robocop: off=too-long-test-case
-Verify Pipeline Tasks Run On GPU Nodes Only When Tolerations Are Added
+Verify Pipeline Tasks Run On Nvidia GPU Nodes Only When Tolerations Are Added
     [Documentation]    Runs a pipeline that tests GPU availability according to GPU tolerations in pipeline tasks:
     ...    - One task should not have GPUs available, as we don't add the GPU tolerations
     ...    - Another task should have GPUs available, as we add the GPU tolerations
     [Tags]    Tier1    Resources-GPU    NVIDIA-GPUs
 
+    Verify Pipeline Tasks Run On GPU Nodes Only When Tolerations Are Added    ${PIPELINE_NVIDIA_GPU}
+
+# robocop: off=too-long-test-case
+Verify Pipeline Tasks Run On AMD GPU Nodes Only When Tolerations Are Added
+    [Documentation]    Runs a pipeline that tests GPU availability according to GPU tolerations in pipeline tasks:
+    ...    - One task should not have GPUs available, as we don't add the GPU tolerations
+    ...    - Another task should have GPUs available, as we add the GPU tolerations
+    [Tags]    Tier1    Resources-GPU    AMD-GPUs
+
+    Verify Pipeline Tasks Run On GPU Nodes Only When Tolerations Are Added    ${PIPELINE_AMD_GPU}
+
+
+*** Keywords ***
+# robocop: off=too-long-test-case
+Verify Pipeline Tasks Run On GPU Nodes Only When Tolerations Are Added
+    [Documentation]    Runs a pipeline that tests GPU availability according to GPU tolerations in pipeline tasks:
+    ...    - One task should not have GPUs available, as we don't add the GPU tolerations
+    ...    - Another task should have GPUs available, as we add the GPU tolerations
+    [Arguments]     ${pipeline_package_path}
+
     # robocop: off=unused-variable
     ${pipeline_id}    ${pipeline_version_id}    ${pipeline_run_id}    ${experiment_id}=
     ...    DataSciencePipelinesBackend.Import Pipeline And Create Run
     ...    namespace=${PROJECT}    username=${TEST_USER.USERNAME}    password=${TEST_USER.PASSWORD}
     ...    pipeline_name=pytorch-verify-gpu-availability
     ...    pipeline_description=Verifies GPU availability in tasks when using tolerations
-    ...    pipeline_package_path=${PIPELINE_GPU_AVAILABILITY_FILEPATH}
+    ...    pipeline_package_path=${pipeline_package_path}
     ...    pipeline_run_name=pytorch-verify-gpu-availability-run
 
     DataSciencePipelinesBackend.Wait For Run Completion And Verify Status
     ...    namespace=${PROJECT}    username=${TEST_USER.USERNAME}    password=${TEST_USER.PASSWORD}
-    ...    pipeline_run_id=${pipeline_run_id}    pipeline_run_timeout=240
+    ...    pipeline_run_id=${pipeline_run_id}    pipeline_run_timeout=600
```

Comment:
Is this change needed? The pipeline usually runs very fast, but if there are no GPUs available (e.g. because they are used by another pod) we want it to fail soon